src/templates/zh/2019/methodology.html

```diff
@@ -0,0 +1,430 @@
+        </p>
+    </section>
+
+    <section>
+        <h3 id="websites"><a href="#websites" class="anchor-link">网站</a></h3>
+
+        <p>
+          数据集中有 5,790,700 个网站。 其中，移动网站5,297,442 个，桌面网站4,371,973个。大多数网站都包括在移动和桌面子集中。
+        </p>
+
+        <p>
+          HTTP Archive 从 <a href="#chrome-ux-report">Chrome用户体验 报告</a>中获取其网站的URL。Chrome用户体验报告是谷歌的公共数据集，汇集了数百万Chrome用户活跃访问的网站的用户体验。这给了我们一个最新的网站列表，并反映了真实的web使用情况。Chrome用户体验报告数据集包括一个因子系数维度，我们使用它来获取桌面或移动用户访问的所有网站。
+        </p>
+
+        <p>
+          Web Almanac使用的2019年七月的HTTP Archive爬取，是利用了最新可用的Chrome用户体验报告版本 2019 5月 (201905) 作为它的网站列表。 该数据集于2019年6月11日发布，收集了Chrome用户在5月份访问的网站。
+        </p>
+
+        <p>
+          由于资源限制， HTTP Archive 只能对每个网站测试一个在Chrome UX报告之中的页面。为了兼顾这一点，只包括主页。请注意，这将引入一些偏见的结果，因为一个主页不一定代表整个网站。
+        </p>
+
+        <p>
+          HTTP Archive 也被认为是一个实验室测试工具，这意味着它从数据中心测试网站，而不是从真实的用户体验收集数据。因此，所有网站主页都是在一个处于登出的空缓存状态进行测试的。
+        </p>
+    </section>
+
+    <section>
+      <h3 id="metrics"><a href="#metrics" class="anchor-link">指标</a></h3>
+
+      <p>
+        HTTP Archive 收集关于如何构建web的指标。它包括一些基本指标，如每个页面的字节数、页面是否通过HTTPS加载，以及单个请求和响应头。这些指标中的大多数是由<a href="#webpagetest">WebPageTest</a>提供, 它作为每个网站的测试运行工具。
+      </p>
+
+      <p>
+        其他测试工具用于提供关于页面的更高级的指标。 例如， <a href="#lighthouse">Lighthouse</a> 被用来对页面进行审计，以分析它在可访问性和SEO等方面的质量。下面的 <a href="#tools">工具</a>小节将更详细地介绍这些工具。
+      </p>
+
+      <p>
+        为了解决实验室数据集的一些固有限制，Web Almanac 还利用了 <a href="#chrome-ux-report">Chrome UX Report</a> 来度量用户体验，特别是在Web性能方面。
+      </p>
+
+      <p>
+       有些指标是完全无法达到的。例如，我们不一定有能力检测用于创建网站的工具。如果一个网站是使用create-react-app构建的，我们可以看出它使用的是React框架，但不一定是使用了特定的构建工具。除非这些工具在网站代码中留下可探测的指纹，否则我们无法测量它们的使用情况。
+      </p>
+
+      <p>
+        其他的指标不一定是不可能度量的，但是是具有挑战性的或者不可靠的。例如，web设计的某些方面本质上是可视化的，可能很难量化，比如页面是否有插入式模态对话框。
+      </p>
+    </section>
+
+    <section>
+      <h2 id="tools"><a href="#tools" class="anchor-link">工具</a></h2>
+
+      <p>
+        Web Almanac 是在以下开源工具的帮助下实现的。
+      </p>
+    </section>
+
+    <section>
+      <h3 id="webpagetest"><a href="#webpagetest" class="anchor-link">WebPageTest</a></h3>
+
+      <p>
+        <a href="https://www.webpagetest.org/">WebPageTest</a> 是一个卓越的web性能测试工具，同时是HTTP Archive的骨干。我们使用一个WebPageTest的 <a href="https://github.com/WPO-Foundation/webpagetest-docs/blob/master/user/Private%20Instances/README.md">私有实例</a> ，携带着私有的测试UA，它是测试每个web页面的实际浏览器。 桌面和移动网站在不同的配置下进行测试:
+      </p>
+      <div class="table-wrap"><div class="table-wrap-container">
+      <table>
+        <thead>
+          <tr>
+              <th>Config</th>
+              <th>Desktop</th>
+              <th>Mobile</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Device</td>
+            <td>Linux VM</td>
+            <td>Emulated Moto G4</td>
+          </tr>
+
+          <tr>
+            <td>User Agent</td>
```

Comment:
Should User Agent be translated? Or fine to leave as is as it's a technical term?